examples/dt/neutron/3cont_2comp_3net-ipv4-geneve-composable_provider_network

```diff
@@ -0,0 +1,66 @@
+# Deployed Topology 3cont_2comp_3net-ipv4-geneve-composable-vlan-provider-network
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+| Networker                                  | vm    | 3      |
+| CFME tester                                | vm    | 1      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, VLAN                | Must have          |
+|           | performance                  |                    |
+| Glance    | default                      | Must have          |
+| Cinder    | cinder_backup                | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+|           | neutron_azs_ovn              |                    |
+
+
+### Additional information 
+composable_roles
```

Comment:
composable_roles is a tripleo term, we are using it in this job mainly for testing neutron with standalone networker nodes